.ko.yaml

```diff
@@ -0,0 +1 @@
+defaultBaseImage: cgr.dev/chainguard/bash:latest
```

Comment:
there's no kubectl in this image

Reply:
The problem with the chainguard kubectl image is they have set the `kubectl` as the `entrypoint` so while running as docker container it is very problematic. 
So i believe if user needs `kubectl` for some of it's operation it can easily install it before hand using script.

Reply:
Did you try it out ?

Reply:
Yes I tried this out it's a problem. 
it just depends whether you are using `pod` or `docker containers`
If you talk about pod then we no issues if someone is using docker then there is a issue

Reply:
Let me check I am not sure if after the build the entypoint is overwritten